projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageDeployer.java

```diff
@@ -42,6 +42,9 @@ public class JobImageDeployer {
   @Value("${datajobs.deployment.readOnlyRootFilesystem:}")
   private boolean readOnlyRootFilesystem;
 
+  @Value("${datajobs.deployer.imagePullPolicy:IfNotPresent}")
```

Comment:
Let's use the same prefix as the above property `datajobs.deployment.jobImagePullPolicy` like `datajobs.deployment.readOnlyRootFilesystem`